sir_cake/seller/views.py

```diff
@@ -163,6 +168,14 @@ def _mark_order_as_shipped(request, order):
         messages.success(request, 'Order marked as shipped')
 
 
+def _get_url_name_for_navbar(request):
+    url = request.resolver_match.url_name
+    if 'item' in url:
+        return 'seller_all_items'
+    if 'order' in url:
+        return 'seller_orders'
```

Comment:
It it intentional that this function returns `None` if none if statements are executed? Even if it is, I would still add an explicit `return None` at the end because it shows explicitly what the correct logic is.